src/custom/state/price/reducer.ts

```diff
@@ -56,7 +57,8 @@ function initializeState(
 function getResetPrice(sellToken: string, buyToken: string, kind: string) {
```

Comment:
```suggestion
function getResetPrice(sellToken: string, buyToken: string, kind: OrderKind) {
```

Reply:
Okok, let me do this and some rearangements in another PR, cause i think this comes from a bad modeling we did.

Reply:
#902